cfb-mode/src/decrypt.rs

```diff
@@ -78,6 +155,18 @@ where
     }
 }
 
+impl<C> IvState for BufferDecryptor<C>
+where
+    C: BlockEncryptMut + BlockDecrypt + BlockCipher,
+{
```

Comment:
I think implementing `IvState` could be misleading. You will not be able to restore a mode state by passing retrieved "IV" into the `InnerIvInit` trait. I think it would be better to add inherent methods, which could look roughly like this:
```rust
fn get_state(&self) -> (&Block<C>, usize) { .. }
fn restore_state(cipher: C, block: &Block<C>, pos: usize) -> Self { .. }
```

Reply:
How would you reconstruct `cipher` in this case? I failed to do find a way to do that, and I can't pull it out because of the `Drop` impl

Reply:
It has to be constructed by user manually using a case-specific approach. It's similar to how `IvState` return only IV without returning an underlying cipher.

These methods imply serialization of the state and we can not generically serialize cipher state, i.e. user could work with `&Aes128`, but he would have to save or submit key used for its initialization, not the reference.

Reply:
thanks, fixed it up accordingly